### DISCUSSION TITLE
2D or 3D monsters/enemies

### DISCUSSION THREAD

**Discussion by kodriq** [jeffaroundtheworld/2024gamecollab#14] [Polls]
vote on which you would prefer, see the art/graphics discussion for more details on each, but basically

**3D** = we use premade 3D models for the game (from Mixamo or smth)
**2D** = we make our own unique 2D models and adapt them to fit a 3D game

_(there's like a 1% chance we could make our own 3D models but that would be more of a "final project" ambition)_

**Comment by kodriq** [comment]
FYI if its just me drawing i cant promise they would be the most amazingly crafted works of art ever for the 2D models but they could be uniquely ours

**Reply by Rhyswatsonn** [reply]
i can help with this too if you want

**Reply by kodriq** [reply]
that would be awesome

**Comment by jeffaroundtheworld** [comment]
I think the conclusion was reached to do 2D style enemies.
